packages/accounts-controller/CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [17.0.0]
+
+### Changed
+
+- **BREAKING:** Newly added account is no longer set as the last selected account ([#4363](https://github.com/MetaMask/core/pull/4363))
+- Bump `@metamask/eth-snap-keyring` to `^4.3.1` ([#4405](https://github.com/MetaMask/core/pull/4405))
+- Bump `@metamask/keyring-api` to `^8.0.0` ([#4405](https://github.com/MetaMask/core/pull/4405))
+- Bump `@metamask/keyring-controller` to `^17.1.0` (`devDependencies`) ([#4413](https://github.com/MetaMask/core/pull/4413))
```

Comment:
Hmm we generally don't include devDeps bumps in the changelog, but I'm not sure why keyring-controller isn't a dependency. 

That might be a holdover from a past state when keyring-controller only supplied types to accounts-controller?

@mcmire Should we recategorize keyring-controller as a dependency here?

Reply:
You're right, this should be moved to dependency now. To not block this release, i'll update this in another PR

Reply:
Sounds good!

Reply:
Oh weird my comment got added in the wrong place. Well you got the idea :)